client/src/components/EventCard.tsx

```diff
@@ -114,7 +100,24 @@ export const EventCard: React.FC<EventCardProps> = ({ event }) => {
             area={'eventstart'}
             marginBottom={['1', '2']}
           >
-            Starting: {formatDate(event.start_at)}
+            {event.canceled ? (
+              <Text
+                data-cy="event-canceled"
+                color="red.500"
+                fontSize={['smaller', 'sm']}
+                fontWeight={'semibold'}
+              >
+                Canceled: {formatDate(event.start_at)}
+              </Text>
+            ) : new Date(event.start_at) < new Date() ? (
+              <Text fontSize={['smaller', 'sm']} fontWeight={'semibold'}>
+                Passed: {formatDate(event.start_at)}
+              </Text>
+            ) : (
+              <Text fontSize={['smaller', 'sm']} fontWeight={'semibold'}>
+                Upcoming: {formatDate(event.start_at)}
+              </Text>
+            )}
```

Comment:
I've played around a bit with this part.

Under the `metaTag`:
```ts
  const eventStatus = event.canceled
    ? 'Canceled'
    : new Date(event.start_at) < new Date()
    ? 'Passed'
    : 'Upcoming';
  const canceledStyle = { 'data-cy': 'event-canceled', color: 'red.500' };
```
And then:
```suggestion
            <Text
              {...(event.canceled && canceledStyle)}
              fontSize={['smaller', 'sm']}
              fontWeight={'semibold'}
            >
              {eventStatus}: {formatDate(event.start_at)}
            </Text>
```